### DISCUSSION TITLE
x-transition in x-bind Alpine 3.2

### DISCUSSION THREAD

**Discussion by jacksleight** [alpinejs/alpine#1690] [7. Bugs (archived)]
Version 3.2.0 introduced a fix (https://github.com/alpinejs/alpine/pull/1637) for issues #1633 and #1458, which allowed using functions in `x-bind` object transition properties. However, in my testing these still aren't working in 3.2. I have created a demo here with two examples, one using strings and one using functions:

https://codepen.io/jacksleight/pen/WNjbExq?editors=1001

When toggling test 2 no transitions are applied.

**Comment by jacksleight** [comment]
Just wondering if anyone had any ideas on this? I'm still struggling to get transition functions working, not sure if I'm doing it wrong or there's a bug? Thanks.

**Comment by calebporzio** [comment]
@jacksleight - thanks for the sample code - I used it to fix this issue and tagged a new release with the fix: v3.2.2

Here's the PR of the fix if you care: https://github.com/alpinejs/alpine/pull/1785

**Reply by jacksleight** [reply]
Brilliant, thanks!
